unleash-yggdrasil/src/strategy_parsing.rs

```diff
@@ -990,4 +1022,50 @@ mod tests {
 
         std::env::remove_var("hostname");
     }
+
+    #[test_case("127.0.0.1", "127.0.0.1", true; "Exact match")]
```

Comment:
All of these are shamelessly stolen from the Java SDK test cases, which is the only SDK that deals with IPv6 and subnets